src/importlinter/contracts/layers.py

```diff
@@ -39,19 +39,29 @@ class LayersContract(Contract):
 
     Configuration options:
 
-        - layers:         An ordered list of layers. Each layer is the name of a module relative
-                          to its parent package. The order is from higher to lower level layers.
-        - containers:     A list of the parent Modules of the layers (optional).
-        - ignore_imports: A set of DirectImports. These imports will be ignored: if the import
-                          would cause a contract to be broken, adding it to the set will cause
-                          the contract be kept instead. (Optional.)
+        - layers:             An ordered list of layers. Each layer is the name of a module
+                              relative to its parent package. The order is from higher to lower
+                              level layers.
+        - containers:         A list of the parent Modules of the layers (optional).
+        - ignore_imports:     A set of DirectImports. These imports will be ignored: if the import
+                              would cause a contract to be broken, adding it to the set will cause
+                              the contract be kept instead. (Optional.)
+        - exhaustive:         If true, check that the contract declares every possible layer in
+                              its list of layers to check. This option cannot be used without the
+                              containers option. (Optional.)
+        - exhaustive_ignores: A set of potential layers to ignore in exhaustiveness checks. The
```

Comment:
I think it makes for a better API if `exhaustive_ignores` doesn't imply `exhaustive` - means there's just one way of telling if a contract is exhaustive.

We should handle the presence of a `exhaustive_ignores` but no `exhaustive` either by just ignoring the `exhaustive_ignores` or raising `InvalidContractOptions` - I don't feel strongly which.